assets/users/components/EditUser.jsx

```diff
@@ -45,7 +45,7 @@ function EditUserComponent({
         Object.keys(seats[companyId] || {}) :
         products.map((product) => product._id);
     const sections = companyId != null ?
-        companySections[companyId] || [] :
+        (isUserAdmin(user) ? allSections : companySections[companyId] || []) :
```

Comment:
This is only when editing a user. We need to fix the UserProfile so that existing admins can view all sections in the UserProfile (without requiring to edit their user to add this section)

Reply:
you mean here ?
![image](https://github.com/superdesk/newsroom-core/assets/73937490/5498f2fc-62cf-4144-91bf-74dd1a190cc5)

Reply:
Yes

Reply:
got it